cmd/operator-sdk/add/api.go

```diff
@@ -102,8 +106,15 @@ func apiRun(cmd *cobra.Command, args []string) error {
 		Repo:           projutil.CheckAndGetProjectGoPkg(),
 		AbsProjectPath: absProjectPath,
 	}
-
 	s := &scaffold.Scaffold{}
+
+	// Check if any package files for this API group dir exist, and if not
+	// scaffold a stub.go to prevent erroneous gengo parse errors.
```

Comment:
```suggestion
	// scaffold a group.go to prevent erroneous gengo parse errors.
```

Also below `stub`->`group`.